App.js

```diff
@@ -1,9 +1,363 @@
 import React, { useState, useCallback } from 'react';
-import { StyleSheet, Text, View, FlatList, Dimensions, TouchableOpacity} from 'react-native';
+import { StyleSheet, Text, View, FlatList, Dimensions, TouchableHighlight } from 'react-native';
 import { newBoard } from './newboard.js';
 
 export default function App() {
-  
+  const [{ roll, even, odd, position, message, optMessage, score, board }, setGameState] = useState({
+    roll: 0,
+    even: false,
+    odd: false,
+    position: 0,
+    message: 'Roll again',
+    optMessage: 'Kim',
+    score: 0,
+    board: JSON.parse(JSON.stringify(newBoard)),
+  });
+
+  // Nav bar
+  const nav = [
+    { key: 1, name: 'PLAY', invisible: false },
+    { key: 2, name: '', invisible: true },
+    { key: 3, name: '', invisible: true },
+    { key: 4, name: '', invisible: true },
+    { key: 5, name: '', invisible: true },
+    { key: 6, name: 'Score' },
+    { key: 7, name: 0 },
+  ];
+
+  const numColumns = 7;
```

Comment:
Let's make this a constant outside of the component function.

Reply:
DONE (Not sure why you wanted this.)